The typelib for PyGObject gets installed outside of the default
typelib search path, so it is never found unless Liferea is run with
`GI_TYPELIB_PATH=/usr/lib/liferea/girepository-1.0`
This places the typelib in the normal path of /usr/lib/girepository-1.0
<https://github.com/lwindolf/liferea/issues/1458>

diff --git a/src/Makefile.am b/src/Makefile.am
index 2caa32993f7..4e94fc5f337 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -142,7 +142,7 @@ if HAVE_INTROSPECTION
 girdir = $(datadir)/liferea/gir-1.0
 gir_DATA = $(INTROSPECTION_GIRS)
 
-typelibdir = $(libdir)/liferea/girepository-1.0
+typelibdir = $(libdir)/girepository-1.0
 typelib_DATA = $(INTROSPECTION_GIRS:.gir=.typelib)
 
 CLEANFILES = \
